libvast/src/system/default_application.cpp

```diff
@@ -35,16 +35,30 @@
 #include "vast/system/pcap_writer_command.hpp"
 #endif
 
+#define READER(name)                                                           \
+  reader_command<format::name ::reader, defaults::import ::name>, #name
```

Comment:
Removing the whitespace works for me. Is this a gcc workaround?
Same on the other macros.

Reply:
This is simply how clang-format does it.